x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/time_range_step/time_range.tsx

```diff
@@ -18,7 +18,13 @@ import { EventRateChart } from '../charts/event_rate_chart';
 import { LineChartPoint } from '../../../common/chart_loader';
 import { TimeRangePicker } from './time_range_picker';
 import { GetTimeFieldRangeResponse } from '../../../../../services/ml_api_service';
+import { mlJobService } from '../../../../../services/job_service';
+import { ml } from '../../../../../services/ml_api_service';
 
+export interface Duration {
```

Comment:
Wondering if this would be better named `TimeRange`, to avoid confusion with moment.js durations which are generally a number and a string e.g. `6h`?